csrankings-j.csv

```diff
@@ -2187,7 +2187,7 @@ Jun Wei 0001,Chinese Academy of Sciences,http://people.ucas.ac.cn/~weijun,2GVRKY
 Jun Xiao 0001,Zhejiang University,http://mypage.zju.edu.cn/junx,fqOwFhQAAAAJ
 Jun Xu 0001,Renmin University of China,http://info.ruc.edu.cn/academic_professor.php?teacher_id=169,su14mcEAAAAJ
 Jun Xu 0014,Georgia Institute of Technology,http://www.cc.gatech.edu/~jx,mHwonDEAAAAJ
-Jun Xu 0024,Stevens Institute of Technology,https://web.stevens.edu/facultyprofile/?id=2321,9nHXbTEAAAAJ
+Jun Xu 0024,University of Utah,https://sites.google.com/view/junxzm/home

```

Comment:
Missing Google Scholar entry. Web page still says Stevens affiliation. Please point to Utah affiliation.